common/screenshotter.go

```diff
@@ -91,24 +90,19 @@ func (s *screenshotter) fullPageSize(p *Page) (*Size, error) {
                     document.body.clientHeight, document.documentElement.clientHeight
                 ),
             };
-        }`))
+        }`)
 	if err != nil {
 		return nil, err
 	}
-	v, ok := result.(goja.Value)
-	if !ok {
-		return nil, fmt.Errorf("unexpected type %T", result)
+	var size Size
+	if err := convert(result, &size); err != nil {
```

Comment:
This little `convert` helper was very helpful instead of trying to parse `map[string]any` and all of its elements.